junit5-extension/src/main/java/io/kroxylicious/testing/kafka/clients/CloseableAdmin.java

```diff
@@ -126,8 +126,18 @@
 import org.apache.kafka.common.quota.ClientQuotaAlteration;
 import org.apache.kafka.common.quota.ClientQuotaFilter;
 
+/**
+ * The type Closeable admin.
```

Comment:
```suggestion
 * Provides a simple wrapper around a Kafka Admin client to redirect `close()` so that it has a sensible timeout and can thus be safely used in a try-with-resources block. All other methods delegate to the wrapped Admin client. 
```